### DISCUSSION TITLE
My Antmedia Server crashed

### DISCUSSION THREAD

**Discussion by syed-MTalha** [ant-media/Ant-Media-Server#6579] [Q&A] [answered]
My hardware with VMs, including antmedia license activated vm, is crashed. Please let me know how can I reactivate the license on another machine, as I am unable to recover some vms, including antmedia.

Your early response will be highly appreciated

**Comment by mekya** [comment]
Hi @syed-MTalha,
You can use your license in any instance. Just a reminder, license switch takes about 10-15 mins. 

I mean if your license is for a single instance,  you can use the same license in another instance after 10-15 mins you stopped the first instance. 

Let us know if you have further questions.

Cheers
Oguz

**Reply by syed-MTalha** [reply, marked as answer]
Thanks, Oguz. I will do it right away
